### DISCUSSION TITLE
S3 root size ?

### DISCUSSION THREAD

**Discussion by Balbaroo** [iterate-ch/cyberduck#15266] [Q&A] [answered]
Hi,

I’d like to know if it’s possible to see the size of _/_ from a S3 storage.
I have no issue with calculating and seeing the current size of the folder named _default_ – I guess it’s more a bucket than a folder. But is it even possible to see the size of the upper level?

Context: I’ve contacted the support of my S3 storage due to a discrepancy between the disk space left shown on their side and what I see from mine, and they asked me for a screenshot of the _/_ size in Cyberduck after I showed them the _default_ size.

Thanks for any enlightenment.

**Comment by tatze96** [comment, marked as answer]
The size of the mounted root can't be calculated and displayed as the Info window can only be opened on files or folders or, respectively, buckets.

So, to answer your question, it isn't possible to see the size of the upper level or root within Cyberduck.

**Reply by Balbaroo** [reply]
Thanks for your reply. Yes, I noticed that the _Get Info_ button was otherwise greyed out, hence my trouble with the support request.
So I understand there’s no way to fulfil it; thanks for your confirmation.
